packages/sdk-core/src/formatUnitAmount.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Remove the unnecessary decimals such as '.000' that comes from `<Balance>.toHuman()`
+ * @param amountWithUnit eg: '100.0000 SDN'
+ * @returns '100 SDN'
+ */
+export const formatUnitAmount = (amountWithUnit: string): string => {
```

Comment:
I think this function is a bit strange. Why not convert `BN` value to string with proper formatting?

ex: 10000000000000000000 -> "10 SDN"

Reply:
@niklabh Thank you for working on this. 
You can remove this function from the library and I'll remove this function from our portal as well. I can fetch the amount and parse by `BN.toString()`